sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2656,6 +2656,17 @@ object SQLConf {
       .checkValue(_ > 0, "The difference must be positive.")
       .createWithDefault(4)
 
+  val METADATA_CACHE_TTL =
+    buildConf("spark.sql.metadataCacheTTL")
```

Comment:
nit:
```
  val METADATA_CACHE_TTL = buildConf("spark.sql.metadataCacheTTL")
      .doc("...
```

Reply:
Fixed in https://github.com/apache/spark/pull/28852/commits/7d94f943496fb463f18cd445ccbde3f5225c8e7b